.github/workflows/test-workflow.yml

```
@@ -0,0 +1,68 @@
+name: Test PyPI Publish Workflow
+
+on:
```

```suggestion
on:
  workflow_dispatch:
```
We could add a workflow_dispatch trigger for easier testing